src/widgets/TaskCards/SubTaskCard.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, memo } from 'react';
+import {
+    Box, Card, SelectChangeEvent, Typography,
+} from '@mui/material';
+import { StatusEnum, SubTask, tasksActions } from '../../entities/Task';
+import { useAppDispatch } from '../../shared/lib/hooks/redux';
+import { getDateAndTime } from '../../shared/lib/helpers';
+import { SelectStatus } from '../SelectStatus/SelectStatus';
+
+interface SubTaskCardProps {
+    task: SubTask;
+}
+
+export const SubTaskCard: FC<SubTaskCardProps> = memo(({ task }) => {
+    const dispatch = useAppDispatch();
+
+    const handleChange = (e: SelectChangeEvent) => {
+        dispatch(tasksActions.changeSubTaskStatus({ ...task, status: e.target.value as StatusEnum }));
+    };
+
+    return (
+        <Card
+            variant="outlined"
+            sx={{
+                width: '100%',
+                display: 'flex',
+                padding: '10px',
+                alignItems: 'center',
+                justifyContent: 'space-between',
+            }}
+        >
+            <SelectStatus value={task.status} onChange={handleChange} />
+            <Typography width="100%" align="left">
+                {task.title}
+            </Typography>
+            <Box>
+                { task.completedAt && (
+                    <Typography variant="caption">
+                        Done:
+                        {getDateAndTime(task.completedAt)?.date}
```

Comment:
let's show date and time for tasks and sub-tasks

Reply:
Done